src/main/java/org/breedinginsight/brapps/importer/services/processors/GermplasmProcessor.java

```diff
@@ -285,12 +285,6 @@ public void getExistingBrapiData(List<BrAPIImport> importRows, Program program)
             throw new ValidatorException(validationErrors);
         }
 
-        // Check for bad breeding methods
```

Comment:
Was this just left over from when the switch to using ValidatorException happened?

Reply:
Yep!